utils/rpms/daos.spec

```diff
@@ -182,20 +182,25 @@ rpath_files="utils/daos_build.py"
 rpath_files+=" $(find . -name SConscript)"
 sed -i -e '/AppendUnique(RPATH=.*)/d' $rpath_files
 
+%define conf_dir %{_sysconfdir}/daos
+
 scons %{?no_smp_mflags}    \
       --config=force       \
       USE_INSTALLED=all    \
+      CONF_DIR=%{conf_dir} \
       PREFIX=%{?buildroot}
 
 %install
-scons %{?no_smp_mflags}              \
-      --config=force                 \
-      install                        \
-      USE_INSTALLED=all              \
-      PREFIX=%{?buildroot}%{_prefix}
+scons %{?no_smp_mflags}               \
+      --config=force                  \
+      --install-sandbox=%{?buildroot} \
+      %{?buildroot}%{_prefix}         \
+      %{?buildroot}%{conf_dir}        \
```

Comment:
To be clear, you are passing three arguments to `--install-sandbox`?  How does it know that?

Reply:
No. These are all arguments to scons. The two lines after --install-sandbox are build targets. The "install" alias that works for dev tree builds doesn't work here because of how scons does dependency resolution.

Reply:
Oh yes.  I see now.